rust/exchange_rate/src/frontend/src/components/Chart.svelte

```diff
@@ -0,0 +1,251 @@
+<script>
+  // agent to connect to backend canister
+  import { backend_dummy } from "../../../declarations/backend_dummy/index.js";
```

Comment:
@r-birkner could you please check in your local `backend_dummy/index.js`? We will replace it with `exchange_rate/index.js`.

Reply:
You can create it yourself by running `dfx generate`. In the file, you will then have to replace the condition in line 19 with `true` to make it work for now.

Reply:
Thanks @r-birkner , that worked.